### ISSUE TITLE
Modularize jsog

### ISSUE THREAD

**Issue by nlisker** [jsog/jsog-jackson#22]
Can this project be turned into a java module? This will help when using tools like jdeps and jlink. I believe there are no downsides.

**Comment by stickfigure** [comment]
Would that require targeting Java9 and abandoning Java8?

**Comment by nlisker** [comment]
No, module-info.class will just be ignored on pre-9. Since this project uses Mavan, there is an explanation [here](https://maven.apache.org/plugins/maven-compiler-plugin/examples/module-info.html). Looking at the current pom, I see that it targets even Java 6, so look further down in the link.
